addon/system/model/errors.js

```diff
@@ -214,19 +233,35 @@ export default Ember.ArrayProxy.extend(Ember.Evented, {
     @method add
     @param {String} attribute
     @param {(Array|String)} messages
+    @deprecated
```

Comment:
From the conversation, it seems, these methods were not meant to be deprecated. Should this be undone?